docs/static/bundle.js

```diff
@@ -811,7 +811,9 @@ var defaultOptions = {
     focus: false,
     focusManagement: null,
     hostSelector: '.expander__host',
-    hover: false
+    hover: false,
+    ariaHostSelector: null,
+    expandedClass: null
```

Comment:
Should the default ariaHostSelector not be `expander-expanded`? Come to think of it, I prefer `expander__host-expanded` now. All in on BEM :-)

Reply:
No, `.expander__host-expanded` is only applied to the thing handling the `click`, not the aria. And, that class is _only_ applied when the aria-expanded is not applied to the item that was clicked.

Unless you think the class should always accompany the change?

Reply:
Gotcha. Maybe it stays as null, but is always applied to the hostSelector if present? Might be less confusing that way.

Reply:
So, you're saying:
```
if (this.options.ariaHostSelector) {
    this.hostEl.classList.add(this.options.expandedClass);
}
```
instead of:
```
if (this.options.expandedClass) {
    this.hostEl.classList.add(this.options.expandedClass);
}
```
... or are you saying we remove this boolean check?

Reply:
```js
if (this.options.expandedClass) {
    this.hostEl.classList.add(this.options.expandedClass);
}
```

Reply:
Okay, that's the way it was. Leaving it be.